src/proposals/20260115/GroveEthereum_20260115.t.sol

```diff
@@ -0,0 +1,276 @@
+
+        // 1e18 shares, 1.15e6 max expected assets
+        assertEq(controller.maxExchangeRates(Ethereum.GROVE_X_STEAKHOUSE_USDC_MORPHO_VAULT), 1.15e24);
+
+        assertEq(controller.maxSlippages(Ethereum.CURVE_RLUSD_USDC),   0.9990e18);
+        assertEq(controller.maxSlippages(Ethereum.AAVE_CORE_RLUSD),    0.9990e18);
+        assertEq(controller.maxSlippages(Ethereum.AAVE_CORE_USDC),     0.9990e18);
+        assertEq(controller.maxSlippages(Ethereum.AAVE_HORIZON_RLUSD), 0.9990e18);
+        assertEq(controller.maxSlippages(Ethereum.AAVE_HORIZON_USDC),  0.9990e18);
+    }
+
+    function test_ETHEREUM_onboardCctpTransfersToBase() public onChain(ChainIdUtils.Ethereum()) {
+        bytes32 generalCctpKey  = MainnetController(MAINNET_NEW_CONTROLLER).LIMIT_USDC_TO_CCTP();
+        bytes32 baseCctpKey = RateLimitHelpers.makeDomainKey(
+            MainnetController(MAINNET_NEW_CONTROLLER).LIMIT_USDC_TO_DOMAIN(),
+            CCTPv2Forwarder.DOMAIN_ID_CIRCLE_BASE
+        );
+
+        _assertUnlimitedRateLimit(generalCctpKey); // Set in the GroveEthereum_20250807 proposal
+        _assertRateLimit(baseCctpKey, 0, 0);
+
+        assertEq(MainnetController(MAINNET_NEW_CONTROLLER).mintRecipients(CCTPv2Forwarder.DOMAIN_ID_CIRCLE_BASE), bytes32(0));
+
+        executeAllPayloadsAndBridges();
+
+        _assertUnlimitedRateLimit(generalCctpKey);
+        _assertRateLimit(baseCctpKey, MAINNET_CCTP_RATE_LIMIT_MAX, MAINNET_CCTP_RATE_LIMIT_SLOPE);
+
+        assertEq(
+            MainnetController(MAINNET_NEW_CONTROLLER).mintRecipients(CCTPv2Forwarder.DOMAIN_ID_CIRCLE_BASE),
+            CastingHelpers.addressToCctpRecipient(Base.ALM_PROXY)
+        );
+    }
+
+    function test_ETHEREUM_offboardAgoraAusd() public onChain(ChainIdUtils.Ethereum()) {
+        bytes32 mintKey = RateLimitHelpers.makeAssetDestinationKey(
+            MainnetController(MAINNET_NEW_CONTROLLER).LIMIT_ASSET_TRANSFER(),
+            Ethereum.USDC,
+            MAINNET_AGORA_AUSD_USDC_MINT_WALLET
+        );
+
+        executeAllPayloadsAndBridges();
+
+        _assertZeroRateLimit(mintKey);
+
+        GroveLiquidityLayerContext memory ctx = _getGroveLiquidityLayerContext();
+
+        vm.startPrank(ctx.relayer);
+        MainnetController(ctx.controller).mintUSDS(1e12);
+        MainnetController(ctx.controller).swapUSDSToUSDC(1);
+        vm.expectRevert("RateLimits/zero-maxAmount");
+        MainnetController(ctx.controller).transferAsset(Ethereum.USDC, MAINNET_AGORA_AUSD_USDC_MINT_WALLET, 1);
+        vm.stopPrank();
+
+    }
+
+    function test_BASE_governanceDeployment() public onChain(ChainIdUtils.Base()) {
+        _verifyForeignDomainExecutorDeployment({
+            _executor : Base.GROVE_EXECUTOR,
+            _receiver : Base.GROVE_RECEIVER,
+            _deployer : DEPLOYER
+        });
+        _verifyArbitrumReceiverDeployment({
+            _executor : Base.GROVE_EXECUTOR,
+            _receiver : Base.GROVE_RECEIVER
+        });
```

Comment:
This is somewhat confusing since the Optimism receiver was deployed:

```suggestion
        _verifyOptimismReceiverDeployment({
            _executor : Base.GROVE_EXECUTOR,
            _receiver : Base.GROVE_RECEIVER
        });
```